src/client.js

```diff
@@ -19,6 +19,12 @@ export class UiClient {
         this.defaultI18n = defaultI18n;
     }
 
+    /**
+     *
+     * @param options `option.redirectUrl` is where the confirmation email will redirect.
+     *                `option.returnToAfterEmailConfirmation` is where the page after the conformation email will redirect.
```

Comment:
```suggestion
     *                `option.returnToAfterEmailConfirmation` is the page to which the confirmation email redirects. This parameter is supported by the hosted page "/email-confirmation".
```

Reply:
`the page which the confirmation email redirects to.` 😄

Reply:
```suggestion
     *                `option.returnToAfterEmailConfirmation` the URL to which the user is redirected to following email confirmation.
```